Automattic/wp-calypso
# RTS: Pattern Library reposition and text changes

Related to:
- https://github.com/Automattic/dotcom-forge/issues/8163

## Proposed Changes

* Created 2 new themes for the pattern Section component, `blue` and `gray`
* Repositioned some elements on the Pattern Library to better convey the value.
* Updated some titles and descriptions on the Pattern Library.

## Why are these changes being made?


* To provide a better explanation of the differences between a Readymade template, a Pattern, and a Page Layout.

## Testing Instructions



* Open the Calypso live link
* visit `/patterns`
* You should see the new design and texts.

## Pre-merge Checklist
